src/pages/docs/providers/gpu/gpu-golem-live.md

```diff
@@ -0,0 +1,288 @@
+
+Learn more at:
+[➜ golem-gpu-live @ GitHub](https://github.com/golemfactory/golem-gpu-live)
+
+## 1. Download and unzip image
+
+In the first step, download and unzip the `gpu-golem-live` provider image.
+
+To do this, click on the link below or copy the address and paste it in your web browser. The download will begin.
+
+[Download image](https://mirror.notset.fr/golem/golem-gpu-live-231215T091610Z.img.xz)
+
+```bash
+https://gpu-provider.dev.golem.network/golem-gpu-live-latest.img.xz
+```
+
+Alternatively, you can download the file using a terminal by typing the command:
+
+```bash
+wget https://gpu-provider.dev.golem.network/golem-gpu-live-latest.img.xz
+```
+
+The file takes about 2 GB and should download in about 3 minutes (on a 100 Mbps connection).
+
+Once the download is complete, unzip the file with any tool (such as the default archiving tool, The Unarchiver or 7-Zip) or use the command prompt:
+
+```
+gunzip golem-gpu-live-latest.img.xz
+```
+
+Done! The unzipped file should take up about 8.6 GB.
+
+## 2. Record image to SSD
+
+In the second step, you need to burn the unzipped `gpu-golem-live` image to an external SSD. To do this, you can use free software (such as balenaEtcher) or a command in the terminal.
+
+You can use dd to save the extracted image to the SSD. Connect the SSD to your computer and then check its path on your system using `sudo fdisk -l` (Linux) or `diskutil list` (macOS). Assuming the SSD is referred to on the system as `/dev/disk4`, and the image file (named `golem-gpu-live-latest.img`) is in the `/Downloads` directory you can use:
+
+```bash
+sudo dd if=Downloads/golem-gpu-live-latest.img of=/dev/disk4
+```
+
+Alternatively, you can use the previously mentioned [balenaEtcher](https://etcher.balena.io/#download-etcher) tool.
+Download the appropriate software version for your system and run the wizard.
+
+1.  **Select a file from disk** - click on the `Flash from file` button to select a previously unzipped `gpu-golem-live` image file.
+
+    ![Flash from file](/gpu/balena_1.png)
+
+2.  **Select SSD** - click on the `Select target` button to select the SSD, which is the place where the `gpu-golem-live` image will be installed.
+
+    ![Select target 1](/gpu/balena_2.png)
+    ![Select target 2](/gpu/balena_3.png)
+
+3.  **Record the image!** - Click on the `Flash!` button to start the recording process
+
+    ![Flash!](/gpu/balena_4.png)
+
+    ⚠️ For larger capacity SSDs, you may see additional confirmation - click `Yes, I'm sure`.
+
+    ![Yes, I'm sure](/gpu/balena_5.png)
+
+4.  **Recording** - in the last step you need to be patient. Recording the video, depending on the speed of the SSD, will take from 5 to 10 minutes. While recording, you will be able to watch the progression in real time.
```

Comment:
Recording sounds super wrong, and I don't think ive ever seen it used before for this purpose. The correct term would be flashing

Reply:
ok -> Flashing